README.md

```diff
@@ -1844,6 +2170,71 @@ CreditNote.Log log = CreditNote.Log.get("5155165527080960");
 System.out.println(log);
 ```
 
+## Credit Preview
+You can preview different types of credits before creating them (Currently we only have CreditNote previews):
+
+### Create a CreditNotePreview
+You can preview a Credit Note before creation of the CCB contract:
+
+```java
+import com.starkinfra.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+HashMap<String, Object> sac = new HashMap<>();
+sac.put("type", "sac");
+sac.put("nominalAmount", 100000L);
+sac.put("scheduled", getDateString(3));
+sac.put("taxId", "20.018.183/0001-80");
+sac.put("initialDue", getDateString(4));
+sac.put("nominalInterest", 15);
+sac.put("count", 1);
+sac.put("interval", "year");
+sac.put("rebateAmount", 0L);
+
+HashMap<String, Object> price = new HashMap<>();
+price.put("type", "price");
+price.put("nominalAmount", 100000L);
+price.put("scheduled", getDateString(3));
+price.put("taxId", "20.018.183/0001-80");
+price.put("initialDue", getDateString(4));
+price.put("nominalInterest", 15);
+price.put("count", 15);
+price.put("interval", "year");
+price.put("rebateAmount", 900L);
+
+HashMap<String, Object> american = new HashMap<>();
+american.put("type", "american");
+american.put("nominalAmount", 100000L);
+american.put("scheduled", getDateString(3));
+american.put("taxId", "20.018.183/0001-80");
+american.put("initialDue", getDateString(4));
+american.put("nominalInterest", 15);
+american.put("count", 5);
+american.put("interval", "month");
+american.put("rebateAmount", 900L);
+
+HashMap<String, Object> bullet = new HashMap<>();
+bullet.put("type", "bullet");
+bullet.put("nominalAmount", 100000L);
+bullet.put("scheduled", getDateString(3));
+bullet.put("taxId", "20.018.183/0001-80");
+bullet.put("initialDue", getDateString(4));
+bullet.put("nominalInterest", 15);
+bullet.put("rebateAmount", 0L);
+
+List<CreditPreview> creditNotePreviews = new ArrayList<>();
```

Comment:
chama isso de preview tmb, é nosso padrão:

`List<CreditPreview> previews = new ArrayList<>();`
Sempre que você for dar um nome pra variável com mais de uma palavra, olha o padrão pra ver se ta cert porque muito provavelmente não vai estar